### PR TITLE
Add currency and OSM admin definitions for Tonga and Fiji

### PR DESCRIPTION
**Pull request by olivierdalang**
### What does it fix?

* Ticket: no ticket
* Funded by: SPC
* Description: 
  * Adds currency and OSM admin definitions for Tonga and Fiji

### Checklist:
- [x] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature -> too small of a feature for changelog
- [ ] Unit test for new code added -> no new code added
- [x] Request someone to review or test your PR

### CODE REVIEW

File: safe/resources/osm/admin_level_per_country.json [hunk 1]
```diff
@@ -171,5 +171,36 @@
...
+	  "4": "Division",
+	  "5": "",
+	  "6": "",
+	  "7": "",
+	  "8": "",
+	  "9": "",
+	  "10": ""
+	}
   }
 }
```
Comment (ismailsunni) [inline safe/resources/osm/admin_level_per_country.json]: New line at the end of the file.

**Comment by ismailsunni** [comment]
Hi @olivierdalang 
Can you update it according to my trivial comment above?
And can you make a PR also to branch `inasafe_4`? We will use the `inasafe_4` branch to bug fix release, and I think this PR can be included to it.
We will use develop branch for the new version 5 (for QGIS 3.2)

Thanks!

File: safe/definitions/currencies.py [hunk 2]
```diff
@@ -42,10 +42,26 @@
...
+    'name': 'Tongan Pa\'anga',
+    'short name': 'Pa\'anga',
+    'symbol': u'TS$'
+}
+
+fjd = {
+    'key': 'FJD',
+    'name': 'Fijian Dollar',
+    'short name': 'Dollar',
+    'symbol': u'FJ$'
```
Comment (Gustry) [inline safe/definitions/currencies.py]: develop is now in python 3 so no need for the `u` unicode in both symbols

File: safe/resources/osm/admin_level_per_country.json [hunk 3]
```diff
@@ -171,5 +171,36 @@
...
+	  "6": "",
+	  "7": "",
+	  "8": "",
+	  "9": "",
+	  "10": ""
+	}
+  },
+  "Fiji": {
+	"bbox": [[176.73,-15.37,180,-20.16],
+	         [-180,-15.37,-178.10,-20.16]],
```
Comment (Gustry) [inline safe/resources/osm/admin_level_per_country.json]: "180" "-180": nice :-) crossing this longitude

**Comment by olivierdalang** [comment]
Ok I made the adjustments.

@ismailsunni For inasafe 4, should I target inasafe_4 or master ?

**Comment by ismailsunni** [comment]
> @ismailsunni For inasafe 4, should I target inasafe_4 or master ?

@olivierdalang to `inasafe_4`

**Pull request merged**